src/dfx/src/lib/builders/motoko.rs

```diff
@@ -146,6 +145,19 @@ impl CanisterBuilder for MotokoBuilder {
         };
         motoko_compile(&self.logger, cache.as_ref(), &params)?;
 
+        // Generate stable types
```

Comment:
Do we still need a separate build step for this (and --idl). It should all get produced with the main compilation to binary, if you add the flags.

Reply:
Right, I haven't tried the new binary yet. We can fix this when SDK bumps the new moc release.